INSTALL_STORE_CONTAINER.md

```diff
@@ -65,27 +65,12 @@ The container installation method requires Docker 1.10+ running on 64-bit linux.
 
 The node container (or plugin) should be installed on each Docker node where you want to consume StorageOS volumes or to present capacity to other nodes.
 
-### Network Block Device (NBD)
+### Linux-IO (LIO)
 
-(Optional) NBD is a default Linux kernel module that allows block devices to be run in userspace. Enabling NBD is recommended as it will increase performance for some workloads. To enable the module and increase the number of allowable devices, you must either run:
+StorageOS is developed using Linux-IO (LIO) Target. An open-source implementation of the SCSI target.
+However, LIO is supported by most of the kernels available nowadays, some distributions have left the kernel module out of the main kernel package.
 
-```bash
-$ sudo modprobe nbd nbds_max=1024
-```
-
-**To ensure the NBD module is loaded on reboot.**
-
-1. Add the following line to `/etc/modules`
-
-```
-nbd
-```
-
-2. Add the following module configuration line in `/etc/modprobe.d/nbd.conf`
-
-```
-options nbd nbds_max=1024
-```
+Enabling LIO is mandatory for the best performance, check how to enable LIO here <https://docs.storageos.com/docs/reference/os_support>.
```

Comment:
Remove "for the best performance".

Reply:
Thank you! Done https://github.com/storageos/storageos.github.io/pull/148/commits/85f8b90c9002e67efe6eff881dff4b73ed98fad3